tests/dune.inc

```diff
@@ -4546,7 +4546,7 @@
  (alias spec-649)
  (action (diff spec-649.html spec-649.html.new)))
 (alias
- (name runtest)
+ (name test-specs)
```

Comment:
Why this change?

Reply:
I assume we want `dune (test|runtest)` to run all the tests for the packages bundled in this repository, so we don't want `runtest` to only be assigned to this this set of tests. Thus, this suite is renamed, so we I could define the special `runtest` alias to point to both suites.

Reply:
This can be done without renaming this alias: `alias` stanzas are not "exclusive", there may be many `alias` stanzas attaching dependencies to the same alias.